packages/lesswrong/client/logging.js

```diff
@@ -47,6 +50,8 @@ addCallback("router.onUpdate", ({oldLocation, newLocation}) => {
   captureEvent("navigate", {
     from: oldLocation.pathname,
     to: newLocation.pathname,
+    fromPostId: oldLocation.params?._id,
```

Comment:
This won't work if the post is part of a sequence and using the sequence URL or a collection URL.

Reply:
Hmm. Good to know. Will want to fix that.